spec/asyncapi.md

```diff
@@ -460,83 +399,71 @@ The following shows how multiple servers can be described, for example, at the A
 ```yaml
 servers:
   development:
-    url: development.gigantic-server.com
-    description: Development server
+    host: localhost:5672
+    description: Development AMQP broker.
     protocol: amqp
-    protocolVersion: 0.9.1
+    protocolVersion: 0-9-1
     tags:
       - name: "env:development"
-        description: "This environment is meant for developers to run their own tests"
+        description: "This environment is meant for developers to run their own tests."
   staging:
-    url: staging.gigantic-server.com
-    description: Staging server
+    host: rabbitmq-staging.in.mycompany.com:5672
+    description: RabbitMQ broker for the staging environment.
     protocol: amqp
-    protocolVersion: 0.9.1
+    protocolVersion: 0-9-1
     tags:
       - name: "env:staging"
-        description: "This environment is a replica of the production environment"
+        description: "This environment is a replica of the production environment."
   production:
-    url: api.gigantic-server.com
-    description: Production server
+    host: rabbitmq.in.mycompany.com:5672
+    description: RabbitMQ broker for the production environment.
     protocol: amqp
-    protocolVersion: 0.9.1
+    protocolVersion: 0-9-1
     tags:
       - name: "env:production"
-        description: "This environment is the live environment available for final users"
+        description: "This environment is the live environment available for final users."
 ```
 
-The following shows how variables can be used for a server configuration:
+
+#### <a name="serverObject"></a>Server Object
+
+An object representing a message broker, a server or any other kind of computer program capable of sending and/or receiving data. This object is used to capture details such as URIs, protocols and security configuration. Variable substitution can be used so that some details, for example usernames and passwords, can be injected by code generation tools.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="serverObjectHost"></a>host | `string` | **REQUIRED**. The server host name. It MAY include the port. This field supports [Server Variables](#serverObjectVariables). Variable substitutions will be made when a variable is named in `{`braces`}`.
+<a name="serverObjectProtocol"></a>protocol | `string` | **REQUIRED**. The protocol this server supports for connection.
+<a name="serverObjectProtocolVersion"></a>protocolVersion | `string` | The version of the protocol used for connection. For instance: AMQP `0.9.1`, HTTP `2.0`, Kafka `1.0.0`, etc.
```

Comment:
In one of examples you wrote (this example in the `Server Variable`) the `path` field, did you do it on purpose or did you forget to remove it?

Reply:
Ah! true! Forgot to add path here 🤦 Thanks for the pointer.